src/components/comment/_comment_event.scss

```diff
@@ -2,74 +2,56 @@
 
 .euiCommentEvent {
   overflow: hidden;
+}
 
-  .euiCommentEvent__header {
-    @include euiBreakpoint('xs', 's') {
-      margin: -($euiSizeXS);
-
-      > div {
-        margin: $euiSizeXS;
-      }
-
-      .euiCommentEvent__headerTimestamp {
-        padding-top: $euiSizeXS * .5;
-      }
-    }
-
-    align-items: center;
-
-    .euiCommentEvent__headerData > div {
-      padding-right: $euiSizeXS;
-    }
-  }
+.euiCommentEvent__header {
+  line-height: $euiLineHeight;
+  display: flex;
+  align-items: center;
+}
 
-  .euiCommentEvent__headerData {
-    align-items: center;
-    display: flex;
-    flex-wrap: wrap;
-  }
+.euiCommentEvent__headerData {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
 
-  .euiCommentEvent__headerUsername {
-    font-weight: $euiFontWeightSemiBold;
+  > div {
+    padding-right: $euiSizeXS;
   }
+}
 
+.euiCommentEvent__headerUsername {
+  font-weight: $euiFontWeightSemiBold;
 }
 
 .euiCommentEvent--regular {
   border: $euiBorderThin;
 
   .euiCommentEvent__header {
-    @include euiBreakpoint('xs', 's') {
-      padding: $euiSizeS;
+    min-height: $euiSizeXXL;
+    background-color: $euiColorLightestShade;
+    border-bottom: $euiBorderThin;
+    padding: $euiSizeXS $euiSizeS;
 
-      /**
-       * Fix for IE when using align-items:center in an item that has min-height
-            (https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042)
-       */
+    /**
+     * Fix for IE when using align-items:center in an item that has min-height
+        (https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042)
+     */
+    // sass-lint:disable-block mixins-before-declarations
+    @include internetExplorerOnly {
       &::after {
         content: '';
-        min-height: inherit;
+        // Calculates the minimum height based on full header's min-height minus the vertical padding
+        min-height: $euiSizeXXL - $euiSizeS;
```

Comment:
Basically the fix for IE creates a new pseudo element "child" flex-item and applies the minimum height to that child to force the height we want. But since it's nested within the `__header` which has top and bottom padding, if we don't subtract that padding, it increases the overall min-height. 

<img width="1151" alt="Image 2020-04-06 at 5 00 22 PM" src="https://user-images.githubusercontent.com/549577/78604855-74b0f300-7828-11ea-8e85-6461c3a74b91.png">


Sometimes it's good to try to understand what the CSS fix is doing so you know better how to tweak it to your implementation.